### DISCUSSION TITLE
Show original file names if enabled

### DISCUSSION THREAD

**Discussion by Korgen1** [diced/zipline#564] [Ideas]
When uploading a file, if keep original name is selected it would be nice if they could show up in the dashboard as the original name, currently it is very difficult to tell what a file is when using random links

Example:
![image](https://github.com/diced/zipline/assets/65197237/ab2c8f8d-4317-469b-bf75-b9f92c3f0b83)

As visible here its really hard to tell what a file is based on the randomized name, it would be nice if we could keep the random URL while having an option to show original file names in the dashboard

**Comment by matejdro** [comment]
Now that V4 is out, this would be nice to have. Ability to use original name everywhere in the UI (on the lists, on top of the file cards etc.), but still keep the random name for URLs only.

**Reply by Rabenherz112** [reply]
Agreed, also the option to search after your original file name would be awesome.

**Reply by foorschtbar** [reply]
Yes, i miss that too. Random name only for links and keep the original name on every place in the UI.

**Comment by matejdro** [comment]
@diced would you be open to a PR for this?
